runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process/actions/zip.ts

```diff
@@ -1,11 +1,15 @@
 import englishZipTranslations from '../../../en/process/actions/zip';
 
-const zipTranslations: typeof englishZipTranslations = {
-    'Process.Details.Modeler.Actions.Zip.UnzipFile.Label': 'Rozpakuj archiwum',
-    'Process.Details.Modeler.Actions.Zip.Unzip.Path': 'Ścieżka',
-    'Process.Details.Modeler.Actions.Zip.Unzip.Name': 'Nazwa',
+const zipTranslations: typeof englishZipTranslations  = {
+    'Process.Details.Modeler.Actions.Zip.Unzip.Label': 'Rozpakuj archiwum',
+    'Process.Details.Modeler.Actions.Zip.Path': 'Ścieżka',
+    'Process.Details.Modeler.Actions.Zip.Name': 'Nazwa',
     'Process.Details.Modeler.Actions.Zip.Unzip.Path.Info': 'Ścieżka do folderu zawierającego plik zip, np. "C:\\Users\\User\\Documents".',
-    'Process.Details.Modeler.Actions.Zip.Unzip.Name.Info': 'Nazwa pliku (bez rozszerzenia), np. "zippedArchive".'
+    'Process.Details.Modeler.Actions.Zip.Unzip.Name.Info': 'Nazwa pliku (bez rozszerzenia), np. "zippedArchive".',
```

Comment:
if i got it right then it should be sth like:
"Opcjonalna nazwa pliku (bez rozszerzenia), np. "zippedArchive". Jeżeli nie zostanie podana, spakowany zostanie folder pod ścieżką"

Reply:
no, for unzip folder name is mandatory - this is how Erwin did id, so path is not mandatory and filename is mandatory
for zipping file it is the other way around and it makes sense when you think of it.
I won't change it since this is ok

Reply:
<img width="133" alt="image" src="https://github.com/runbotics/runbotics/assets/69820069/7a6df1c0-dba3-42c9-9487-76cdc839940c">
<img width="215" alt="image" src="https://github.com/runbotics/runbotics/assets/69820069/081f395f-9d5d-423b-bd5d-4ab1966b0b2f">
I'm feeling bamboozled

Reply:
so how does the zip action works?
1. Zip file: 
[filename]: filename.jpg
[path]: path/to/folder
2. Zip folder:
[filename]: folder
[path]: path/to/

Reply:
Imo this should work like this:
1. Zip file:
[filename]: filename.jpg
[path]: path/to/folder
2. Zip folder:
[filename]: ""
[path]: path/to/folder